e3sm_to_cmip/lib.py

```diff
@@ -592,10 +587,17 @@ def get_dimension_data(filename, variable, levels=None, get_dims=False):
                     'hybm': ds['hybm'],
                     'hybi': ds['hybm'],
                 })
+            elif name =="sdepth":
+                data.update({
+                    "levgrnd": ds["levgrnd"].values,
+                    "levgrnd_bnds": get_levgrnd_bnds()
```

Comment:
Takes into account `sdepth` level and maps the data to the correct key in the Dataset object.